RadarSDK/Include/Radar.h

```diff
@@ -533,7 +534,13 @@ completionHandler:(RadarSendEventCompletionHandler)completionHandler NS_SWIFT_NA
  @see https://radar.com/documentation/trip-tracking
  */
 + (void)startTripWithOptions:(RadarTripOptions *_Nonnull)options
-           completionHandler:(RadarTripCompletionHandler _Nullable)completionHandler NS_SWIFT_NAME(startTrip(options:completionHandler:));
+           completionHandler:(RadarTripCompletionHandler _Nullable)completionHandler
+    NS_SWIFT_NAME(startTrip(options:completionHandler:)) __deprecated_msg("Use +[Radar startTripWithOptions:trackingOptions:completionHandler:] instead.");
+
```

Comment:
@jrtibbetts Needs docs. Let's make them consistent with Android and incorporate this feedback in both places: https://github.com/radarlabs/radar-sdk-android/pull/236/files#r988873895 @corypisano

Reply:
Added docs.